californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -237,6 +240,12 @@ public void onTimeout() {
 				public void onCancel() {
 					clearBlock1Status(key);
 				}
+
+				@Override
+				public void onSendError(Throwable error) {
+					copyToken();
+					clearBlock1Status(key);
+				}
```

Comment:
I'm not sure you understand the point. (or maybe it's me :p)
I would like to know if we should add a `copyToken` for the `onSent` event ? (not onSendError)